server/views/temporary-accommodation/assessments/show.njk

```diff
@@ -29,7 +29,11 @@
           <div class="moj-button-menu">
             <div class="moj-button-menu__wrapper">
               {% for action in actions %}
-                <a href="{{ action.href }}" role="menuitem" draggable="false" class="moj-button-menu__item {{ action.classes }}" data-module="govuk-button" data-secondary="true">
+                {% if action.newTab %}
+                  <a href="{{ action.href }}" role="menuitem" draggable="false" class="moj-button-menu__item {{ action.classes }}" data-module="govuk-button" data-secondary="true" rel="noreferrer noopener" target="_blank">
```

Comment:
I know we didn't discuss this but I think we ought to consider changing the button text to be "Place referral (opens in a new tab)". I know it's long and will cause the actions list to become bigger but [it seems like the right thing to do](https://design-system.service.gov.uk/styles/typography/#opening-links-in-a-new-tab)?

Reply:
Ah yeah, I did see that. It's ugly, but yeah I think you're right!